internal/api/public/api_invitation.go

```diff
@@ -193,7 +193,7 @@ func (a *InvitationApiService) CreateInvitationExecute(r ApiCreateInvitationRequ
 		return localVarReturnValue, nil, &GenericOpenAPIError{error: err.Error()}
 	}
 
-	localVarPath := localBasePath + "/api/api/invitations"
+	localVarPath := localBasePath + "/api/invitations"
```

Comment:
is this intentional? It's not obvious how it's connected to the Makefile changes.

Reply:
Yep. this is generated file.